chatgpt_voice_assistant/command_line_parser.py

```diff
@@ -85,6 +85,7 @@ def parse(self) -> CommandLineArguments:
             log_level=parsed_args.log_level,
             max_tokens=parsed_args.max_tokens,
             open_ai_key=parsed_args.open_ai_key,
+            open_ai_model=parsed_args.open_ai_model,
```

Comment:
Do we have to make a corresponding change to the "chatgpt_voice_assistant/models/command_line_arguments.py" file?

Reply:
I checked your branch and the change is there, but weirdly not appearing in the diff here.